Documentation/install/upgrade.rst

```diff
@@ -9,14 +9,53 @@ Kubernetes Cilium Upgrade
 
 Cilium should be upgraded using Kubernetes rolling upgrade functionality in order to minimize network disruptions for running workloads.
 
-Make sure you are using the latest RBAC role and service account definitions
-before performing the rolling upgrade:
+Once the ``cilium-ds.yaml`` file does not contain the ConfigMap provided by the
+``cilium.yaml``, the safest way to upgrade Cilium to version "\ |SCM_BRANCH|" is
+by updating the RBAC rules and the DaemonSet file provided, which makes sure
+the ConfigMap already stored in the cluster will not be affected by the upgrade.
+Both files are dedicated to "\ |SCM_BRANCH|" for each Kubernetes version.
```

Comment:
I found this paragraph a little confusing. Do we tell people to change the cilium-ds.yaml file somewhere? I would expect it to start with "The safest way to upgrade cilium is by updtaing the RBAC rules and Daemonset from the files provided. These make sure ..."

Reply:
Changed to:

>The safest way to upgrade Cilium to version "\ |SCM_BRANCH|" is by updating the
>RBAC rules and the DaemonSet file provided, which makes sure the ConfigMap,
>initially set up by ``cilium.yaml``, already stored in the cluster will not be
>affected by the upgrade.